src/server/services/wfs3/qgswfs3handlers.h

```diff
@@ -0,0 +1,233 @@
+/***************************************************************************
+                              qgswfs3handlers.h
+                              -------------------------
+  begin                : May 3, 2019
+  copyright            : (C) 2019 by Alessandro Pasotti
+  email                : elpaso at itopen dot it
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGS_WFS3_HANDLERS_H
+#define QGS_WFS3_HANDLERS_H
+
+#include "qgsserverogcapihandler.h"
+
+class QgsServerOgcApi;
+
+/**
+ * The APIHandler class Wfs3handles the API definition
+ */
+class Wfs3APIHandler: public QgsServerOgcApiHandler
```

Comment:
Should a Qgs prefix be added here too?

Reply:
Not sure, they are not part of the API, but I can add it if you think it's desireable, no problem.

Reply:
Considering the fact that we use some classes coming from external libraries (json or inja), I think it could be nice to add the Qgs prefix. This way,  we don't have to think much about the origin :).